x-pack/test/functional/es_archives/task_manager_removed_types/mappings.json

```diff
@@ -0,0 +1,2635 @@
+{
```

Comment:
nit.
I don't think we need the mappings in its entirety - I think you can remove all types that aren't a task.
Just to avoid the huge additional file :)